open-api/rest-catalog-open-api.yaml

```diff
@@ -2774,6 +3062,140 @@ components:
           additionalProperties:
             type: string
 
+    ScanTasks:
+      type: object
+      description:
+        Scan and planning tasks for server-side scan planning
+
+
+        - `plan-tasks` contains opaque units of planning work
+
+        - `file-scan-tasks` contains a partial list of table scan tasks
```

Comment:
Yeah it can technically be a complete list, this can be confused somewhat easily if folks interpret it as "not complete" . So 2 options imo:

1.) Remove "partial". This is clean but I think the problem is folks may interpret just "list of table scan tasks" as always complete (it's kind of the oppposite problem)

2.) Use an alternative word like "subset". This seems better because a subset can either be a partial or complete list.

Reply:
@flyrain thanks for bringing this up. I think I agree that we should clarify this and will go with your recommendation.